src/cplscheme/BiCouplingScheme.cpp

```diff
@@ -148,4 +157,48 @@ void BiCouplingScheme::exchangeInitialData()
   }
 }
 
+void BiCouplingScheme::storeTimeStepSendData(double relativeDt)
+{
+  PRECICE_ASSERT(relativeDt > 0);
+  PRECICE_ASSERT(relativeDt <= 1.0, relativeDt);
+  for (auto &aSendData : getSendData()) {
+    auto theData = aSendData.second->values();
+    aSendData.second->storeDataAtTime(theData, relativeDt);
+  }
+}
+
+void BiCouplingScheme::storeTimeStepReceiveDataEndOfWindow()
+{
+  if (hasDataBeenReceived()) {
+    for (auto &aReceiveData : getReceiveData()) {
+      auto theData = aReceiveData.second->values();
+      aReceiveData.second->overrideDataAtEndWindowTime(theData);
```

Comment:
```suggestion
      receiveData.second->overrideDataAtEndOfWindow(values);
```
Some naming suggestions

Reply:
Done in ced11922ff5ec01e43af3cec9bb8bf191b85d8d1.